src/allmydata/test/test_connection_status.py

```diff
@@ -0,0 +1,122 @@
+"""
+Tests for allmydata.util.connection_status.
+
```

Comment:
Moving connection status tests made viewing the diff a little difficult, but I think this is a good change.